server/backend/src/users/users.controller.ts

```diff
@@ -47,6 +52,41 @@ export class UsersController {
     }
   }
 
+  @Put(':uid/groups/:gid/cameras')
+  async putCamera(
```

Comment:
nit, see my other comment
```suggestion
  async registerCamera(
```